cpython-windows/build.py

```diff
@@ -1327,6 +1327,25 @@ def build_cpython(
         )
 
         if pgo:
+            if (
+                meets_python_minimum_version(python_version, "3.14")
+                and build_platform == "x64"
+            ):
+                # On 3.14.0a5, PGO cannot be run on ceval so we disable it there
+                # See https://github.com/python/cpython/issues/130004
+                # We should be able to drop this patch in 3.14.0a6
+                exec_and_log(
+                    [
+                        "patch",
```

Comment:
Presuming `patch` isn't available here and is why we don't have Windows patchfiles, but we'll see. Will follow with another approach if so.

Reply:
`git apply` might work too if you have git.